humanize/time.py

```diff
@@ -166,3 +166,66 @@ def naturaldate(value):
     return naturalday(value)
 
 
+def english_list(
```

Comment:
I think this should be an internal function:
```suggestion
def _english_list(
```